### PR TITLE
Add 4.6 release config

### PR DESCRIPTION
**Pull request by sjha4**
Adding 4.6 katello config.

### CODE REVIEW

File: configs/katello/4.6.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+  :smart_proxy_container_gateway:
+    :branch: 1.0.6
+    :repo: https://github.com/Katello/smart_proxy_container_gateway.git
+  :katello-selinux:
+    :branch: 4.0.2
+    :repo: https://github.com/Katello/katello-selinux.git
+:ignores:
+:strict_keys: true
+:gpg_key: 2A6A4BAA
```
Comment (sjha4) [inline configs/katello/4.6.yaml:45]: Needs to be updated.

File: configs/katello/4.6.yaml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+:tags:
+  - name: katello-4.6-thirdparty-candlepin-rhel7
+    based_off: katello-thirdparty-candlepin-rhel7
+    build_target: katello-4.6-rhel7-build
+    arches:
```
Comment (sjha4) [inline configs/katello/4.6.yaml:50]: Do we have to remove the el7 config?

Comment (ekohl) [reply configs/katello/4.6.yaml:50]: Yes, also for nightly though not necessarily in this PR.

File: configs/katello/4.6.yaml (hunk 1 above)
Comment (ekohl) [reply configs/katello/4.6.yaml:45]: Normally the release engineer (@pcreech this time) creates the key. There's no reason we can't generate the key earlier so we can move it up in the planning if that makes things easier in the future.

Comment (sjha4) [reply configs/katello/4.6.yaml]: Thanks @ewoud and @pcreech : Updated keys.

**Comment by sjha4** [comment]
Update host-tools to 4.2.0

**Review by pcreech — APPROVED** [review]

**Pull request merged**